src/otx/core/model/entity/base.py

```diff
@@ -229,21 +231,22 @@ def _export_parameters(self) -> dict[str, Any]:
         "This is used in the constructor of `self._exporter`. "
         "For example, `self._exporter = SomeExporter(**self.export_parameters)`. "
         "Please refer to `otx.core.exporter.*` for detailed examples."
-
         Returns:
             dict[str, Any]: parameters of exporter.
         """
         parameters = {}
-
         all_labels = ""
         all_label_ids = ""
         for lbl in self.label_info.label_names:
             all_labels += lbl.replace(" ", "_") + " "
             all_label_ids += lbl.replace(" ", "_") + " "
 
+        # not every model requires ptq_config
+        ptq_config = self.ptq_config if hasattr(self, "ptq_config") else ""
```

Comment:
I'd make the config empty by default instead of checking attributes here.
Also ptq_config can be renamed into optimization_config, since we refer the whole process as "opotimization"

Reply:
Do you propose to add empty PTQ config for all the models? Like adding it as property to OTXModel?

Reply:
> I'd make the config empty by default instead of checking attributes here.
Also, ptq_config can be renamed into optimization_config, since we refer the whole process as "optimization"

Thanks @sovrasov, that's what I want to say.

Reply:
Yes, since this config is common, so we can define it at the top of models hierarchy